mount/mount.c

```diff
@@ -23,6 +30,42 @@ static char* get_fs_type() {
 	return fs_type;
 }
 
+static int fill_ipv4_sockaddr(const char *hostname, struct sockaddr_in *addr)
```

Comment:
I think either a comment, or maybe renaming the function that this is supposed to resolve the destination address would be valuable.

Reply:
Yea honestly I think this code is more complex than we need.
I'll try to simplify this.